cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -391,6 +410,10 @@ func indexRepo(ctx context.Context, gitDir string, sourcegraph Sourcegraph, o *i
 		args = append(args, "-delta_threshold", strconv.FormatUint(o.DeltaShardNumberFallbackThreshold, 10))
 	}
 
+	if o.UseSourcegraphIDForName {
+		args = append(args, "-use_sourcegraph_id_for_name")
+	}
```

Comment:
just to make sure I understand, we make this optional since we need to think about a migration path between MT vs non -MT? That migration path can be tackled later, but for now in MT we can use the "final state"